### ISSUE TITLE
Laravel application not starting (?)

### ISSUE THREAD

**Issue by EmanueleCoppola** [orchestral/testbench-dusk#22]
Hello, I'm developing a Laravel package, I already implemented testbench for Unit tests, now I want to make some Browser tests using orchestra/testbench-dusk.
But I encountered a possible issue.
I pushed the tests code here: https://github.com/SickCRUD/CRUD/tree/master/tests dumping the browser page while testing I get always :
```html
<html xmlns="http://www.w3.org/1999/xhtml"><head></head><body></body></html>
```
I tried with many routes, it looks like the Laravel app is not started. Am I wrong in something? Thanks in advance.

**Comment by Keoghan** [comment]
Hi @EmanueleCoppola, 

I've had a quick look around - I think most of the problem you are experiencing will be fixed with the new release for 3.5 (I just released it).  It should sort #21 too.

Once you composer update and get the new version, screenshots will appear in the right place which will help a bit too.

To get closer to working tests on your test system, I also needed to:

 - rename src/app to src/App
 - move your Auth test to the following namespace:` namespace SickCRUD\CRUD\Tests\Browser\Auth;`
 - change the directory for prepareSqlLite() to `$directory = __DIR__ . '/../vendor/orchestra/testbench-dusk/laravel/database';`

Once that's complete, there are some more issues with NoCaptcha - but you should be well on your way.

Error logs can be found in vendor/orchestra/testbench-dusk/laravel/storage/logs

Hope that helps.

**Comment by EmanueleCoppola** [comment]
Well, I still have some errors.
This is the screeenshow of the output: https://i.gyazo.com/3eef40b9f6854a7022b22ded912f82ad.png
The output of the browser is
```json
[
    {
        "level": "SEVERE",
        "message": "http:\/\/127.0.0.1:8000\/sick-crud\/login - Failed to load resource: net::ERR_CONNECTION_REFUSED",
        "source": "network",
        "timestamp": 1519763722384
    },
    {
        "level": "SEVERE",
        "message": "http:\/\/127.0.0.1:8000\/sick-crud\/login - Failed to load resource: net::ERR_CONNECTION_REFUSED",
        "source": "network",
        "timestamp": 1519763722626
    }
]
```
Laravel logs: empty.

It looks like `server.php` has something wrong, but still I can't figure out what. Maybe the server is not starting...

Since if I keep the php process running with the server active, I can't access to http://localhost:8000 from another browser page...

**Comment by EmanueleCoppola** [comment]
@Keoghan found the error.
In DuskServer.php at line 162, in Win platform "exec" is not needed. I'll make a pull.

**Comment by Keoghan** [comment]
Super, thanks ;)

**Comment by EmanueleCoppola** [comment]
Here are the pulls #25 #24 #23

**Comment by EmanueleCoppola** [comment]
@Keoghan what about make a new release to close this issue?

**Comment by Keoghan** [comment]
Just released on 3.5 branch, will sort out 3.4 a little later today.
